Modem (#6)
hostmobility/documentation: docs/interfaces/modem/gemalto.md

---

> title: Gemalto Pls8 Modem

Reviewer: ```suggestion
title: Gemalto PLS8 Modem
```

---

## Production settings

>     - **In some cases the modem AT command channel is not setup then manually send this commands:**

Reviewer: ```suggestion
    - **In some cases the modem AT command channel is not set up. Do do this manually send the commands below:**
```

        |--> Carrier detected.  Starting PPP immediately.           |
        +-----------------------------------------------------------+
        |--> Starting pppd at Tue Feb  5 10:46:34 2019              |
        +-----------------------------------------------------------+
        |--> Pid of pppd: 600                                       |
        +-----------------------------------------------------------+
        |--> Using interface ppp0                                   |
        +-----------------------------------------------------------+
        |root@mx4-v61:~# --> local  IP address 46.194.12.163        |
        +-----------------------------------------------------------+
        |--> remote IP address 10.64.64.64                          |
        +-----------------------------------------------------------+
        |--> primary   DNS address 195.54.122.211                   |
        +-----------------------------------------------------------+
        |--> secondary DNS address 195.54.122.221                   |
        +-----------------------------------------------------------+

## GPS 
**Note:** simple commands
-------------------------

>     - This is just one way of many ways to setup and get gps position data.

Reviewer: ```suggestion
    - This is just one way of many to set up and get GPS position data.
```
